tests/integration/test_databroker.py

```diff
@@ -80,10 +81,14 @@ def test_databroker_running(self):
         )
         self._start_kafka_cluster()
 
+        os.environ[
+            "PACKAGE_URL"
+        ] = "https://s3-eu-west-1.amazonaws.com/databroker.dev/test-app/MyProducer902.mda"
```

Comment:
Have you thought about the implications of making your development blobstore URL public, and not fronting it with CloudFront?

Please find an alternative.

Reply:
this has been changed to a template value